### PR TITLE
handle @fastify/x names to export

### PR DESCRIPTION
**Pull request by Uzlopak**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by jsumners — COMMENTED** [review]
What is this solving?

**Comment by Uzlopak** [comment]
Before we had e.g. fastify-kafka as module name which was automatically changed to the named export fastifyKafka So it was basically generating `module.exports.fastifyKafka = plugin`

We renamed all fastify own modules to the form @fastify/x-y-z. So now the named exports are '@fastify/xYZ' which is equivalent to `module.exports['@fastify/xYZ'] = plugin`.

If the current behaviour is correct, than we have to rewrite the failing test in https://github.com/fastify/fastify-kafka/pull/69 accordingly. If not, we have to merge this, and then the test in that PR will pass.

**Review by climba03003 — APPROVED** [review]
LGTM.

File: test/bundlers.test.js [hunk 1]
```diff
@@ -70,6 +70,28 @@ test('from kebabo-case to camelCase', (t) => {
...
+    next()
+  }, {
+    name: '@hello/world'
+  })
+
+  t.equal(plugin.helloWorld, plugin)
+  t.end()
+})
+
+test('from @-prefixed named kebabo-case to camelCase', (t) => {
```
Comment (zekth) [inline test/bundlers.test.js:84]:
```suggestion
test('from @-prefixed named kebab-case to camelCase', (t) => {
```

**Review by zekth — CHANGES_REQUESTED** [review]

File: test/bundlers.test.js (hunk 1 above)
Comment (Uzlopak) [reply test/bundlers.test.js:84]: yeah, well. I oriented myself to the test cases above...

**Comment by Uzlopak** [comment]
@zekth you want to review again?
@jsumners do you have further questions?

**Review by zekth — APPROVED** [review]

**Review by Fdawgs — APPROVED** [review]

**Review by Eomm — APPROVED** [review]

**Pull request merged**